text/0083-component.md

```diff
@@ -0,0 +1,118 @@
+# Component
```

Comment:
I like the idea of automatically generating an attribute for the name of the Tracer|Meter, but I'm not sure "component" carries the right meaning, especially after reading the discussion about N:M mappings. A component sounds like a single "part", but this feels more complicated. Was LibraryName not good enough?

Reply:
We were also discussing the term (telemetry) reporter or reporting library on https://github.com/open-telemetry/oteps/pull/76#discussion_r364383941 as another alternative.

Reply:
@jmacd I don't care about the name that much (was proposing one option). I care only about the fact that all telemetry generated by one named "Tracer/Meter" should be annotated with the properties passed to the named "Tracer/Meter".

Reply:
I like the proposal but also agree "Component" doesn't convey the meaning of the thing well. So maybe `InstrumentorInfo`/`InstrumentationInfo`

Reply:
Done.